database/index.js

```diff
@@ -28,8 +28,51 @@ const getAllPhotos = (id, callback) => Photos.find({ product_id: id }, {}, { sor
   }
 });
 
+const savePhotos = (seed, callback) => {
+  
+  let photos = new Photos({
+    product: seed.product,
+    product_id: seed.product_id,
+    index: seed.index,
+    color: seed.color,
+    image_url: seed.image_url,
+  });
+
+  photos.save((err, pictures) => {
+    if(err) return console.error(err);
+    callback(err, pictures)
+  })
+
+}
+
+const updatePhoto = (id, color, callback) => {
+    
+  Photos.findOneAndUpdate({product_id: id}, color, {new: true}, (err, res) => {
+    if(err) {
+      console.log('something went wrong updating data!')
```

Comment:
consider passing the error up the callback chain to make debugging easier if something goes wrong